test/run_pass/test_pwm.js

```diff
@@ -19,12 +19,17 @@ var Pwm = require('pwm');
 
 var pwm = new Pwm();
 
-var configuration = {};
+var configuration = {
+  period: 0.001,  // 1kHz
+  dutyCycle: 0.2  // 20%
+};
 
 if (process.platform === 'linux') {
   configuration.pin = 0;
 } else if (process.platform === 'nuttx') {
   configuration.pin = require('stm32f4dis').pin.PWM1.CH1_1;
+} else if (process.platform === 'tizenrt') {
+  configuration.pin = require('artik053').pin.PWM0;
```

Comment:
I disagree with you, but you are the maintainer here.

Memory usage: This is 6 int props, propably much less space than default config for spi...
Usability: please try to find out where in Artik053 docs there is info about PWM pins, additionally, we will have to provide mapping from raw magic numbers to actual PWM lines, so developer will have to do:
var PWM0 = 0; open({pin: PWM0... 
to avoid hardcoded magic values in the source code. Exactly the same, but will have to read more docs to know the number...